src/arrayMethodJoin.js

```diff
@@ -4,8 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let result = '';
+
+    for (let i = 0; i < this.length; i++) {
+      if (this[i] === null || this[i] === undefined) {
+        result += '';
+      } else {
+        result += `${this[i]}`;
+      }
```

Comment:
```suggestion
      if (this[i] !== null || this[i] !== undefined) {
        result += `${this[i]}`;
      };
```
In this case, when `this[i] === null/undefined` we just do nothing, otherwise - add this[i] to the string